src/App.css

```diff
@@ -1,4 +1,116 @@
 body {
+  background-color: #FFF;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  min-width: 390px;
+}
+
+.container {
   display: flex;
   flex-direction: column;
+  width: 100%;
+  height: 100%;
+  max-width: none;
+  margin: 0 auto;
+}
+
+.img {
+  max-width: 100%;
+  margin-top: 7%;
+  margin-bottom: 3%;
+  margin-left: 7%;
+  margin-right: 7%;
+  border-radius: 40px;
+}
+
+.text {
+  max-width: 100%;
+  padding: 15px;
+  text-align: left;
+  margin-bottom: 5%;
+}
+
+.home h1,
+.home p {
+  color: #242424;
+}
+
+.home p {
+  font-size: 16px;
+  justify-content: start;
+}
+
+.home a {
+  color: #242424;
+  text-decoration: underline;
+}
+
+.home a:hover {
+  color: #007BFF;
+}
+
+@media screen and (min-width: 1250px) and (max-width: 1280px) {
```

Comment:
To ensure your page looks good on all screen sizes, consider adjusting the width and height. On larger screens, having text fill the entire width can be overwhelming for users. Set a maximum width, such as 1280 pixels, and center your content on the page for a better user experience.